fusion_docs/guide.mdx

```diff
@@ -6,14 +6,15 @@ title: User guide
 
 Fusion is a virtual, lightweight, distributed file system designed to optimise the data access of Nextflow data pipelines.
 
-Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler
-pipeline logic and faster, more efficient pipeline execution.
+Fusion enables seamless filesystem I/O to cloud object stores via a standard POSIX interface resulting in simpler pipeline logic and faster, more efficient pipeline execution.
+
+:::note
+Fusion is licensed by Seqera and can be used in Seqera Platform or integrated directly with Nextflow.
```

Comment:
```suggestion
You can try out Fusion for free. To use Fusion in a production environment, a license is required. Fusion is licensed by Seqera for use in Seqera Platform or directly with Nextflow."
```

Reply:
@justinegeffen Production use is a pretty nebulous term here that may cause confusion. Do we know what the exact use limits for the trial will be? Something like "Fusion is free to evaluate for X (time/pipeline runs/GB data/etc.). Thereafter, a license is required to use Fusion in Seqera Platform or Nextflow."

Reply:
Great points, @llewellyn-sl. Thank you. I'm committing my change so it's easier to see where we're at, but will revert on this ASAP.